webapp/sources/rudder/rudder-web/src/main/elm/onboarding/sources/View.elm

```diff
@@ -0,0 +1,220 @@
+import Html.Events exposing (onClick, onInput)
+import List exposing (any, intersperse, map, sortWith)
+import List.Extra exposing (minimumWith)
+import String exposing (lines)
+import Dict exposing (..)
+
+view : Model -> Html Msg
+view model =
+  let
+    completeClass = "section-complete"
+    completeIcon  = "fa fa-check"
+
+    warningClass  = "section-warning"
+    warningIcon   = "fas fa-exclamation"
+
+    defaultClass  = "section-default"
+    defaultIcon   = "fa fa-info"
+
+    visitedClass  = "section-visited"
+
+{--
+    getSelectedMetrics : String
+    getSelectedMetrics =
+      case Dict.get 2 model.sections of
+        Just m -> case m of
+          Metrics _ v -> v
+          _ -> "no"
+        Nothing -> "no"
+--}
+    sidebarSection : Section -> Html Msg
+    sidebarSection s =
+      let
+        titleSection =
+          case s of
+            Welcome _        -> "Welcome"
+            Account _ _      -> "Account"
+            Metrics _ _      -> "Metrics"
+            GettingStarted _ -> "Getting Started"
+
+        sectionIndex = case List.Extra.elemIndex s model.sections of
+          Just i  -> i
+          Nothing -> 0
+        activeClass = if sectionIndex == model.activeSection then "activeSection" else ""
+
+        stateClass  = case s of
+          Welcome _    -> completeClass
+          Account se _ ->
+            case se of
+              Visited  -> visitedClass
+              Complete -> completeClass
+              Warning  -> warningClass
+              _        -> defaultClass
+          Metrics se m ->
+            case se of
+              Visited  -> visitedClass
+              Complete -> completeClass
+              Warning  -> warningClass
+              _        -> defaultClass
+          _ -> defaultClass
+
+      in
+        li [class (activeClass ++ " " ++ stateClass)]
+          [ div [class "timeline-connector"][]
+          , div [class "timeline-item", onClick (ChangeActiveSection sectionIndex)]
+            [ span [class "item-dot"][]
+            , span [class "item-title"][text titleSection]
+            ]
+          ]
+
+    summaryList : Section -> Html Msg
+    summaryList section  =
+      let
+        (stateClass, iconClass, textItem) = case section of
+          Welcome _       -> (completeClass , completeIcon , "Rudder is correctly installed.")
+
+          Account s ac ->
+            case s of
+              Complete -> (completeClass , completeIcon , "Your account "++ ac.username ++ " has been linked to your Rudder!"  )
+              Warning  -> (warningClass  , warningIcon  , "There is a probleme with your account credentials."       )
+              _        -> (defaultClass  , defaultIcon  , "No account have been linked to your Rudder installation." )
+
+          Metrics s m  ->
+            case s of
+              Complete -> (completeClass , completeIcon , "Tech and Core metrics will be shared anonymously with us, thanks for your help!" )
```

Comment:
It should be 
"Complete metrics will be shared anonymously with us, thanks for your help!"
"Minimal metrics will be shared anonymously with us, thanks for your help!"
=> Depending on the value of the metrics